src/utils/triton.h

```diff
@@ -0,0 +1,170 @@
+// The MIT License (MIT)
+//
+// Copyright (c) 2021 NVIDIA CORPORATION
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+
+#ifndef DALI_BACKEND_UTILS_TRITON_H_
+#define DALI_BACKEND_UTILS_TRITON_H_
+
+#include "src/dali_executor/io_descriptor.h"
+#include "src/dali_executor/utils/dali.h"
+#include "triton/backend/backend_model.h"
+#include "triton/backend/backend_model_instance.h"
+
+namespace triton { namespace backend { namespace dali {
+
+
+/**
+ * Converts TRITONSERVER_DataType to dali_data_type_t
+ */
+inline dali_data_type_t to_dali(TRITONSERVER_DataType t) {
+  assert(t >= 0 && t <= 13);
+  // Use the trick, that TRITONSERVER_DataType and dali_data_type_t
```

Comment:
What if TRITON or DALI changes it?

Reply:
I'll change it next time I'll be on this file (it will happen soon), to not inflate this PR further

Reply:
OK